### PR TITLE
Kk devuyst

### PR DESCRIPTION
**Pull request by JossWritesCode**

### CODE REVIEW

File: react-usercard/src/components/UserCard.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+class UserCard extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+     createUserCard() {
```
Comment (JossWritesCode) [inline react-usercard/src/components/UserCard.js:8]: This is interesting. I'm kind of curious why you chose to create the card this way.

File: react-usercard/src/components/UserCard.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return card
+    }
+    render() {
+        var user = this.props.user; 
+        return(
+            <div className="card">
+                <div className="card-info">
+                    <img src={user.avatar_url}></img>
+                    <h3>{user.name}</h3>
+                    <p>{user.login}</p>
```
Comment (JossWritesCode) [inline react-usercard/src/components/UserCard.js:56]: Nice work. How do you intend to display the followers?

File: react-usercard/src/App.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    console.log('Constuctor Invoked!');
+  super();
+  this.state = {
+    userData: {},
+    followers: ""
+    }
+  };
+
+  componentDidMount() {
+    fetch("https://api.github.com/users/kkdev211")
```
Comment (JossWritesCode) [inline react-usercard/src/App.js:17]: great job using fetch!

**Review by JossWritesCode — COMMENTED** [review]
Good work so far. Message me if you need help adding the followers. I'm also interested to chat with you about the function you made to create cards. Nice work!

File: react-usercard/src/components/UserCard.js (hunk 1 above)
Comment (kkdev211) [reply react-usercard/src/components/UserCard.js:8]: I didn't need to- was using this as a reference from my old project. I hadn't cleaned the code up yet.

File: react-usercard/src/components/UserCard.js (hunk 2 above)
Comment (kkdev211) [reply react-usercard/src/components/UserCard.js:56]: I'm re-using the UserCard Component for the followers by mapping over them in the app.js

File: react-usercard/src/App.js (hunk 3 above)
Comment (kkdev211) [reply react-usercard/src/App.js:17]: Thank you :-) Getting the hang of it!

**Pull request merged**